test/unit/tag_test.rb

```diff
@@ -294,4 +294,10 @@ def setup
     assert lat.location_tag?
     assert lon.location_tag?
   end
+
+  test 'Tag.get_recommendations' do
+    nodes = Tag.get_recommendations("blog", "test")
+    assert_not_nil nodes
+    assert nodes.length > 0
```

Comment:
Aha! This assertion failed! So, looks like we have been getting an empty array back:

https://github.com/publiclab/plots2/runs/8217990567?check_suite_focus=true#step:5:85

```
TagTest#test_Tag.get_recommendations (23.30s)
        Expected false to be truthy.
        test/unit/tag_test.rb:301:in `block in <class:TagTest>'
```

Reply:
Great, so now, we can try to fix that. Once we push a fix up as a commit to this PR, the test will start to pass again -- that's actually "Test Driven Development" -- we set up a specific enough test condition (or "assertion") that our test will pass once it's working properly. The test actually precedes the final solution!